### PR TITLE
Add more props to custom CSS loading

### PR DESCRIPTION
**Pull request by pascalwengerter**

### CODE REVIEW

File: src/system.js [hunk 1]
```diff
@@ -14,16 +14,23 @@ contexts.forEach(context => {
   context.keys().forEach(key => components.push(context(key).default))
 })
 
+function initializeCustomProps(tokens, prefix) {
+  for (const param in tokens) {
```
Comment (kulmann) [inline src/system.js:18]: Could you add an early return above the for loop, checking if tokens are provided?

File: src/system.js [hunk 2]
```diff
@@ -14,16 +14,25 @@ contexts.forEach(context => {
   context.keys().forEach(key => components.push(context(key).default))
 })
 
+function initializeCustomProps(tokens, prefix) {
+  if (typeof tokens === "undefined" || Object.keys(tokens).length === 0) {
```
Comment (kulmann) [inline src/system.js:18]:
```suggestion
  if (!tokens) {
```
should be sufficient

Comment (pascalwengerter) [reply src/system.js:18]: Love it!

This only works for nonexistent tokens, if the token object is empty (`spacing: {}`) it doesn't return early but loops for...no content - whatever...

Comment (kulmann) [reply src/system.js]: yep, looping on empty input is fine :-)

**Review by kulmann — APPROVED** [review]
🚀

**Pull request merged**